installer.php

```diff
@@ -0,0 +1,211 @@
+    }
+
+    public function getLatestTag()
+    {
+        if (null !== $this->latestTag) {
+            return $this->latestTag;
+        }
+
+        if (false === $raw = $this->get(sprintf(self::GITHUB_LATEST_RELEASE_URI, self::GITHUB_REPOSITORY_NAME))) {
+            throw new \RuntimeException('Unable to retrieve the latest version');
+        }
+
+        $release = json_decode($raw, true);
+
+        return $this->latestTag = $release['tag_name'];
+    }
+
+    private function get($uri)
+    {
+        return @file_get_contents(
+            $uri,
+            null,
+            stream_context_create(['http' => [
+                'method' => 'GET',
+                'header' => 'User-Agent: Manalize (installer)\r\n',
+            ]])
+        );
+    }
+}
+
+class IO
+{
+    private static $availableForegroundColors = [
+        'red' => ['set' => 31, 'unset' => 39],
+        'cyan' => ['set' => 36, 'unset' => 39],
+        'green' => ['set' => 32, 'unset' => 39],
+        'yellow' => ['set' => 33, 'unset' => 39],
+        'white' => ['set' => 37, 'unset' => 39],
+        'default' => ['set' => 39, 'unset' => 39],
+    ];
+
+    public function write($message)
+    {
+        echo $message;
+    }
+
+    public function writeln($message)
+    {
+        $this->write($message);
+        $this->newLine();
+    }
+
+    public function title($title)
+    {
+        $underline = str_repeat('=', strlen($title));
+        $this->writeColored($title, 'cyan', true);
+        $this->writeColored($underline, 'cyan', true);
+    }
+
+    public function newLine($count = 1)
+    {
+        $this->write(str_repeat(PHP_EOL, $count));
+    }
+
+    public function success($message)
+    {
+        $this->newLine();
+        $this->writeColored("✔ $message", 'green', true);
+    }
+
+    public function error($message)
+    {
+        $this->newLine();
+        $this->writeColored("✘ $message", 'red', true);
+    }
+
+    public function note($message)
+    {
+        $this->writeColored("- $message", 'yellow', true);
+    }
+
+    private function writeColored($message, $color, $newLine = false)
+    {
+        if ($this->hasColorSupport()) {
+            $color = self::$availableForegroundColors[$color];
+            $message = sprintf("\e[%sm%s\e[%sm", $color['set'], $message, $color['unset']);
+        }
+
+        $newLine ? $this->writeln($message) : $this->write($message);
+    }
+
+    /**
+     * @see \Symfony\Component\Console\Output\StreamOuput
+     */
+    protected function hasColorSupport()
+    {
+        if ('\\' === DIRECTORY_SEPARATOR) {
+            return
+                '10.0.10586' === PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD
+                || false !== getenv('ANSICON')
+                || 'ON' === getenv('ConEmuANSI')
+                || 'xterm' === getenv('TERM');
+        }
+
+        return function_exists('posix_isatty') && @posix_isatty(STDOUT);
+    }
+}
+
```

Comment:
Hmm.. now I have some stuff in mind I would like to discuss, let's see that at office.